tests/ert/unit_tests/config/test_ert_config.py

```diff
@@ -1449,22 +1449,14 @@ def run(self, *args):
     assert ert_config is not None
 
 
-@pytest.mark.usefixtures("use_tmpdir")
 def test_validate_job_args_no_warning(caplog, recwarn):
     caplog.set_level(logging.WARNING)
-
-    with open("job_file", "w", encoding="utf-8") as fout:
-        fout.write("EXECUTABLE echo\nARGLIST <ECLBASE> <RUNPATH>\n")
-
-    with open("config_file.ert", "w", encoding="utf-8") as fout:
-        # Write a minimal config file
-        fout.write("NUM_REALIZATIONS 1\n")
-        fout.write("INSTALL_JOB job_name job_file\n")
-        fout.write(
-            "FORWARD_MODEL job_name(<ECLBASE>=A/<ECLBASE>, <RUNPATH>=<RUNPATH>/x)\n"
-        )
-
-    ErtConfig.from_file("config_file.ert")
+    ErtConfig.from_dict(
+        {
+            "INSTALL_JOB": [["name", ("file", "EXECUTABLE echo\nARGLIST <ECLBASE>\n")]],
+            "FORWARD_MODEL": [["name", [("<ECLBASE>=", "A/<ECLBASE>")]]],
+        }
+    )
 
     # Check no warning is logged when config contains
     # forward model step with <ECLBASE> and <RUNPATH> as arguments
```

Comment:
Remove RUNPATH from this comment given the code above.

Reply:
I changed the test to use RUNPATH and removed the comment as it is non-informative in my opinion.